include/benchmark/benchmark_api.h

```diff
@@ -230,7 +230,7 @@ BENCHMARK_UNUSED static int stream_init_anchor = InitializeStreams();
 // expression from being optimized away by the compiler. This function is
 // intended to add little to no overhead.
 // See: https://youtu.be/nXaxk27zwlk?t=2441
-#if defined(__GNUC__)
+#if defined(__GNUC__) && !defined(__pnacl__) && !defined(EMSCRIPTEN)
```

Comment:
`x86_64-unknown-nacl` is (non-Portable) Native Client target, it compiles to regular x86-64 instructions (with some limitations due to sandboxing), and supports inline assembly. You need PNaCl toolchain from Native Client SDK or `emcc` toolchain from Emscripten SDK to try it.